addons/website_sale/static/src/js/sale_variant_mixin.js

```diff
@@ -578,9 +578,9 @@ var VariantMixin = {
             .trigger('change');
 
         $parent
-            .find('.o_product_tags')
-            .first()
-            .html(combination.product_tags);
+            .find('.o_product_tags:first')
+            .html($(combination.product_tags)
+            .html());
```

Comment:
I think this should also do the job and less confusing
```suggestion
            .replaceWith($(combination.product_tags))
```